src/hotspot/share/c1/c1_GraphBuilder.cpp

```diff
@@ -406,7 +410,9 @@ void BlockListBuilder::mark_loops() {
   // a node with some loop bit set that would then propagate back and be never cleared because we would
   // never go back through the original loop header. Therefore if there are any irreducible loops the bits in the states
   // for these loops are going to propagate back to the root.
-  BitMap& loop_state = mark_loops(_bci2block->at(0), false);
+  BlockBegin* start = _bci2block->at(0);
```

Comment:
There is a typo in line 408, that you might want to fix as well `irriducible` -> `irreducible`.